persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java

```diff
@@ -291,6 +296,24 @@ private static NodeList<DocumentMemberDeclarationNode> populateBallerinaNodeList
         return moduleMembers;
     }
 
+    private static NodeList<DocumentMemberDeclarationNode> populatePersistDependency (
+            NodeList<DocumentMemberDeclarationNode> moduleMembers) throws BalException {
+        ClassLoader classloader = Thread.currentThread().getContextClassLoader();
+        try (InputStream inputStream = classloader.getResourceAsStream(BalSyntaxConstants.VERSION_PROPERTIES_FILE)) {
+            Properties properties = new Properties();
+            properties.load(inputStream);
+            moduleMembers = moduleMembers.add(SampleNodeGenerator.createStringKV(BalSyntaxConstants.KEYWORD_GROUP_ID,
+                    BalSyntaxConstants.PERSIST_GROUP_ID, null));
+            moduleMembers = moduleMembers.add(SampleNodeGenerator.createStringKV(BalSyntaxConstants.ARTIFACT_ID,
+                    BalSyntaxConstants.KEYWORD_ARTIFACT_ID, null));
+            moduleMembers = moduleMembers.add(SampleNodeGenerator.createStringKV(BalSyntaxConstants.KEYWORD_VERSION,
+                    properties.get(BalSyntaxConstants.PERSIST_VERSION).toString(), null));
+        } catch (IOException e) {
+            throw new BalException("ERROR: couldn't read the version.properties file. " + e.getMessage());
```

Comment:
```suggestion
            throw new BalException("ERROR: could not read the version.properties file. " + e.getMessage());
```

Reply:
Other error messages also use this format. So I used this https://github.com/ballerina-platform/persist-tools/blob/main/persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java#L188

Reply:
Ack